### ISSUE TITLE
Plugin installation in Linux Fedora 39 freezes right up from start

### ISSUE THREAD

**Issue by Gitomu** [basyura/inkdrop-sidetoc#19]
## Issue Description

I finally made the switch to Linux today and installed Fedora Workstation after a deep research on which one was the most stable and full featured for a Desktop daily use.
After installing and set it up, I downloaded and installed Inkdrop and while the plugins did not sync (not a big deal, I reinstalled them all again); I found out the TOC plugin is not working fine.
It was working great on Windows but I'm not going back that route again, I prefer Linux for my personal machine.

## Repro

1. I install the plugin from within Inkdrop Settings > Install.
2. I access any of my exsiting notes which I know contain headings for sure. 
3. I try to scroll down through the headings in the sidetoc panel and it simply won't move or do anything. 

## Workaround

There's none. Once I uninstall the plugin, things work fine. Unfortunately, disabling it is not an option since the TOC plugin is crashing Inkdrop window for now. I already submitted an issue for that too with the plugin dev.

**Plugin version: 3.1.0**
**Inkdrop version: 5.8.0**

##My Linux Environment

### Hardware Information:

    Modelo de hardware: ASRock B550 Phantom Gaming 4
    Memoria: 32,0 GiB
    Procesador: AMD Ryzen™ 5 4600G with Radeon™ Graphics × 12
    Gráficos: AMD Radeon™ RX 6500 XT
    Capacidad del disco: 2,0 TB

### Software Information

    Versión de firmware: P2.80
    Nombre del SO: Fedora Linux 39 (Workstation Edition)
    Construcción de SO: (null)
    Tipo de SO: 64 bits
    Versión de GNOME: 45.5
    Sistema de ventanas: X11
    Versión del núcleo: Linux 6.7.11-200.fc39.x86_64

**Comment by basyura** [comment]
Since https://github.com/blacroix/inkdrop-toc/issues/3  
is not being maintained, it would be better to post at 
https://github.com/inkdropapp/inkdrop-toc  
or 
https://forum.inkdrop.app/.

Could these be the issues?

* Inkdrop crashes when the TOC Plugin is installed.
* The SideToc Plugin does not track the right header when scrolling through notes.

**Comment by Gitomu** [comment]
Yes, that seems to be the issue I'm experiencing: "Inkdrop crashes when the TOC Plugin is installed."
I will post on the other repository as you suggested. 

Thanks!

**Comment by basyura** [comment]
Can you update to v5.8.1-beta-1?
https://forum.inkdrop.app/t/inkdrop-desktop-v5-8-1-beta-1/4564
